test.restclient

```diff
@@ -0,0 +1,83 @@
+# -*- restclient -*-
```

Comment:
This file looks interesting, but I have no idea how it works!?
Could you enlighten the ignorant :)